### ISSUE TITLE
CircleCI uses Go v1.6.2

### ISSUE THREAD

**Issue by sgerrand** [sgerrand/alpine-pkg-R#17]
This causes `go get github.com/tcnksm/ghr` to fail with

```
package context: unrecognized import path "context" (import path does not begin with hostname)

go get github.com/tcnksm/ghr returned exit code 1

Action failed: go get github.com/tcnksm/ghr
```

https://circleci.com/gh/sgerrand/alpine-pkg-R/66

**Comment by sgerrand** [comment]
Updating the Go installation to version 1.7.x should resolve this issue.

**Comment by sgerrand** [comment]
Fixed in #16.
